src/main/io/osd.c

```diff
@@ -264,6 +269,41 @@ static void osdDrawSingleElement(uint8_t item)
             break;
         }
 
+#ifdef MAG
+    case OSD_HOME_DIR:
+        if (STATE(GPS_FIX) && STATE(GPS_FIX_HOME)) {
+            if (GPS_distanceToHome >0) {
+                int16_t h = GPS_directionToHome - DECIDEGREES_TO_DEGREES(attitude.values.yaw);
+
+                if (h < 0)
+                    h += 360;
```

Comment:
Please always use braces, even when it's just a single statement. This helps avoid errors when code gets extended later on.